chalicelib/app_utils.py

```diff
@@ -1,10 +1,15 @@
+import os
 from os.path import dirname
 from foursight_core.app_utils import AppUtils as AppUtils_from_core
 from .vars import FOURSIGHT_PREFIX, HOST
 
 
 class AppUtils(AppUtils_from_core):
 
+    es_host_from_foursight_core_apply_identity = os.environ.get("ES_HOST")
```

Comment:
Please import and use instead `AppUtilsCore` above. (GitHub won't let me point to that part because it wasn't edited.)

Reply:
Why is this variable called "_from_foursight_core_apply_identity"? It looks like it comes from `os.environ`. Am I missing something?

Reply:
Not sure what you mean about importing/using AppUtilsCore ...
I name that es_host_from_foursight_core_apply_identity just because it is foursight-core/AppUtilsCore which sets that environment (os.environ) variable. Perhaps I should name it more normally (es_host) and just have a comment about it (?)